test/tstDistributedSearchTree.cpp

```diff
@@ -278,6 +278,34 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(one_leaf_per_rank, DeviceType,
                  {}, {0, 0}, {});
 }
 
+BOOST_AUTO_TEST_CASE_TEMPLATE(do_not_exceed_capacity, DeviceType,
+                              ARBORX_DEVICE_TYPES)
+{
+  // This unit tests exposes bug that essentially assumed the number of
```

Comment:
I would remove the history of the bug, and just write what the test does.

Reply:
Now you complain I commented too much?